index.html

```diff
@@ -363,23 +296,23 @@ <h2>2000+ contributors</h2>
 		<div class="row">
 			<div class="col-sm-12 text-center">
 				<h4 class="uppercase mb16">Blogs</h4>
-				<p class="lead mb64">Articles from the contributors</p>
+				<p class="lead mb64">Articles from the students</p>
```

Comment:
same

Reply:
Why this change isn't correct, The blogs are given by students only not by mentors, So if I say it is by contributors it is wrong. Because contributors means students and mentors both.